Formula/embree.rb

```diff
@@ -0,0 +1,45 @@
+class Embree < Formula
+  desc "High-performance ray tracing kernels"
+  homepage "https://embree.github.io/"
+  url "https://github.com/embree/embree/archive/v3.5.0.tar.gz"
+  sha256 "4635439c277d0f688f1a21ad13ad22fdadbcbca04680cce346c88179b9522741"
+  head "https://github.com/embree/embree.git"
+
+  depends_on "cmake" => :build
+  depends_on "ispc" => :build
+  depends_on "tbb"
+
+  def install
+    args = std_cmake_args + %w[
+      -DBUILD_TESTING=OFF
+      -DEMBREE_IGNORE_CMAKE_CXX_FLAGS=OFF
+      -DEMBREE_ISPC_SUPPORT=ON
+      -DEMBREE_MAX_ISA=SSE4.2
```

Comment:
No all macOS-supported machines have SSE4.2. Please check the use of `MacOS.version.requires_sse4` and `MacOS.version.requires_sse42` in our documentation (or other formulas)

Reply:
This is the highest ISA, so a source build could use less. Basically, the bottles need to be built with at least SSE 4 to be useful. What is the CPU in the machine that builds the bottles?

Reply:
So the current version "fixes" the max ISA for instances when SSE4.2 is not supported (to an extent it was redundant since it was "max ISA"), but it does make the formula significantly less useful if the bottles end up being built with only SSE2 support since the OSPRay renderer requires SSE4.2 and that is main reason for using Embree in the first place.